src/components/Header/index.js

```diff
@@ -1,12 +1,121 @@
-import React from 'react'
+import React, { useState } from 'react'
 import PropTypes from 'prop-types'
 import LanguageSwitcher from '../../LanguageSwitcher/LanguageSwitcher'
+// import LocaleContext from '../../context/LocaleProvider'
+import { useStaticQuery, graphql } from 'gatsby'
+import {
+  HeaderContainer,
+  HeaderBg,
+  MenuContainer,
+  TitleContainer,
+  HeaderTitle,
+} from './styles'
+import Title from '../Utilities/Title'
+import MenuLogo from '../../images/Vector.svg'
+import Drawer from '../Drawer'
+import { Tablet, Desktop } from '~/components/Utilities/Media'
+import Icon from '~/components/Icon'
 
 const Header = ({ location }) => {
+  const HeaderQuery = useStaticQuery(graphql`
+    {
+      allCosmicjsHeaders {
+        nodes {
+          metadata {
+            site_sub_title
+            site_title
+            graphic {
+              local {
+                publicURL
+              }
+            }
+            logo {
+              local {
+                publicURL
+              }
+            }
+            logo_desktop {
+              local {
+                publicURL
+              }
+            }
+          }
+        }
+      }
+    }
+  `)
+
+  const [toggleDrawer, setToggleDrawer] = useState(false)
+  const logo =
+    HeaderQuery.allCosmicjsHeaders.nodes[0].metadata.logo.local.publicURL
+
+  const desktopLogo =
+    HeaderQuery.allCosmicjsHeaders.nodes[0].metadata.logo_desktop.local
+      .publicURL
+  const graphic =
+    HeaderQuery.allCosmicjsHeaders.nodes[0].metadata.graphic.local.publicURL
```

Comment:
We'll need to add some safety checks for those object because none are uploaded. We'll run into issues. Like such:

`graphic = HeaderQuery?.allCosmicjsHeaders?.nodes?.[0]?.metadata?.graphic?.local?.publicURL`